### PR TITLE
DOCS-9724 made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/re-order-policies-task.adoc [hunk 1]
```diff
@@ -3,21 +3,23 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Mule Runtime 4.x, which policies run first is configurable with the exception of CORS. CORS has the highest precedence. This procedure describes how to re-order policies provided by Anypoint Platform.
+In Mule runtime engine 4.x, you can configure the order of execution of the policies based on your needs. The only exception to this rule is the CORS policy, which has the highest precedence. 
 
-. In Anypoint Platform, click API Manager, and in API Administration, click the API version or instance.
+To re-order a policy:
```
Comment (kbostrom) [inline modules/ROOT/pages/re-order-policies-task.adoc]:
```suggestion
To reorder a policy:
```

File: modules/ROOT/pages/re-order-policies-task.adoc [hunk 2]
```diff
@@ -3,21 +3,23 @@ ifndef::env-site,env-github[]
...
 
-. On the API dashboard, choose Policies.
+. In Anypoint Platform, select *API Manager > API Administration* and then click the API version or instance.
 
-. In the top of the list of any applied policies, click Edit Policy Order to change the priority.
+. From the left navigation menu, select *Policies*.
+
+. From top of the list of the applied policies, click *Edit Policy Order*""
```
Comment (kbostrom) [inline modules/ROOT/pages/re-order-policies-task.adoc]:
```suggestion
. From the top of the list of the applied policies, click *Edit Policy Order*:
```

**Review by kbostrom — DISMISSED** [review]
CE per request

File: modules/ROOT/pages/re-order-policies-task.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/re-order-policies-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/re-order-policies-task.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/re-order-policies-task.adoc]: Done! Thanks!

**Review by kbostrom — APPROVED** [review]
Reapproved

**Pull request merged**